tests/integration/test_trading.py

```diff
@@ -294,3 +299,795 @@ def test_estimate_position(vega_service_with_market: VegaServiceNull):
+        trading_key=PARTY_C.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_BUY",
+        order_ref="best-ask",
+        price=990,
+        volume=initial_volume,
+        wait=True,
+    )
+    vega.submit_order(
+        trading_key=PARTY_B.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_SELL",
+        price=1200,
+        volume=initial_volume,
+    )
+    vega.submit_order(
+        trading_key=PARTY_A.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_BUY",
+        price=1000,
+        volume=initial_volume,
+    )
+    #AC 0012-NP-LIPE-003: An estimate is obtained for a position with no open volume and a single limit buy order, after the order fills the mark price keeps going down in small increments and the actual liquidation takes place within the obtained estimated range. 
+    estimate_margin_open_vol_only_before_trade, estimate_liquidation_price_including_buy_orders = vega.estimate_position(
+        market_id,
+        open_volume=0,
+        side=["SIDE_BUY"],
+        price=[1000],
+        remaining=[10],
+        is_market_order=[False, False],
+        collateral_available=collateral_available,
+    )
+    print(f"estimate_liquidation_price.best_case.including_buy_orders= {estimate_liquidation_price_including_buy_orders.best_case.including_buy_orders}") 
+    print(f"estimate_liquidation_price.worst_case.including_buy_orders = {estimate_liquidation_price_including_buy_orders.worst_case.including_buy_orders}") 
+
+    vega.submit_order(
+        trading_key=PARTY_B.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_SELL",
+        price=1000,
+        volume=initial_volume,
+    )
+    vega.wait_fn(1)
+    vega.wait_for_total_catchup()
+    
+    market_info = vega.market_info(market_id=market_id)
+    market_data = vega.get_latest_market_data(market_id=market_id)
+    #Check order status/ market state 
+    print(f"{market_info}")
+    print(f"market data before amending at price{market_data.mark_price}= {vega.get_latest_market_data(market_id=market_id)}")
+    print(f"account_PARTY_A = {vega.party_account(key_name=PARTY_A.name, asset_id=asset_id, market_id=market_id)}")
+    print(f"position_PARTY_A = {vega.positions_by_market(key_name=PARTY_A.name, market_id=market_id)}")
+
+    linear_slippage_factor = float(market_info.linear_slippage_factor)
+    quadratic_slippage_factor = float(market_info.quadratic_slippage_factor)
+  
+    estimate_margin_open_vol_only, estimate_liquidation_price_open_vol_only = vega.estimate_position(
+        market_id,
+        open_volume=10,
+        side=["SIDE_BUY"],
+        price=[1000],
+        remaining=[0],
+        is_market_order=[False, False],
```

Comment:
This order doesn't exist on the book any more and instead the party has `open_volume`, lets remove these fields.